### ISSUE TITLE
TV-Categories as own tabs in resource-editing-interface (activatetable in the system-settings)

### ISSUE THREAD

**Issue by JensWolff** [modx-ccc-2015/whishlist#26]
It would be nice, if you could set a system-setting, that allows to divide the template-variables-tab in the resource-editing-interface into different tabs, based on the tv-categories:
![bundle template variables](https://cloud.githubusercontent.com/assets/754205/5792887/9dd72170-9f2d-11e4-86d5-ce504f8a332b.jpg)

**Comment by wuuti** [comment]
+1 - but we have to be careful, as this is "orthogonal" to the already builtin behaviour of showing vertically stacked category tabs for tvs...

**Comment by gpsietzema** [comment]
You can already do this using Form Customization afaik.

**Comment by JensWolff** [comment]
@gpsietzema If you can do this, that would be cool. But how to do it?

**Comment by labr1005** [comment]
@JensWolff 
Manager Customization:

Create a new tab inside "Regions" for your set
![bildschirmfoto-2015-01-21-um-09 56](https://cloud.githubusercontent.com/assets/1913250/5833500/4831e8e4-a154-11e4-8944-fcbfd1de2a6d.jpg)

Set the ID from the new region for your TV
![bildschirmfoto-2015-01-21-um-09 57](https://cloud.githubusercontent.com/assets/1913250/5833504/4ad619bc-a154-11e4-8e6e-bc9389e11609.jpg)

Done
![bildschirmfoto-2015-01-21-um-09 58](https://cloud.githubusercontent.com/assets/1913250/5833505/4d22a910-a154-11e4-919e-6c6a92157d8f.jpg)

**Comment by JensWolff** [comment]
Thanks @labr1005 for the quick Tutorial! Thats a good way to do it! ;-)

**Comment by wuuti** [comment]
Yes, you can do this somehow with FC. But Tabs in FC are buggy (their entries disappear when you deselect them and reload them - problems with articles tabs are known) - and you have to do this manually. I thought @JensWolff wanted the tabs to be automatically created depending on the categories?

**Comment by JensWolff** [comment]
@inreti Okay, I have reopened the issue, so that we perhaps find a better working, automatic solution for this together.

**Comment by rocaoliveira** [comment]
Hi, @labr1005 where i see this option in Modx 2.3.5?
